ingestion/functions/parsing/colombia/colombia.py

```diff
@@ -104,7 +104,7 @@ def parse_cases(raw_data_file, source_id, source_url):
                 "demographics": convert_demographics(
                     entry["Edad"],
                     entry["Sexo"])}
-            if entry["Fecha diagnostico"] != '':
+            if entry["Fecha diagnostico"]:
                 case["events"] = [
```

Comment:
I think this is a great solution, and it should definitely fix the problem.

If you wanted to condense the if/else statement to just the part that it changes, rather than creating the entire events array all at once, you could consider initializing `case["events"]` as an empty list, then creating something like `confirmation_event = {"name": "confirmed"}`, and then just having `if field is there, confirmation_event["dateRange"] = "blah", else confirmation_event["dateRange"] = "other_blah"`.

But this is all entirely opinion and up to you!